addons/addon-base-raas/packages/base-raas-services/lib/schema/update-environment-sc.json

```diff
@@ -16,6 +16,10 @@
       "type": "string",
       "maxLength": 2048
     },
+    "inWorkflow": {
+      "type": "string",
+      "maxLength": 2048
+    },
```

Comment:
nit: A description on what this variable means would be useful.

Reply:
Emmm I can't really add a comment here without ESlint complaining. I searched for ways to add description here but it all seems kinda hacky. Any suggestions?

Reply:
I thought you can just add `"description"` key and the value. Since this is a JSON schema it won't really count towards the actual schema, but just provide extra metadata. Reference: https://json-schema.org/learn/getting-started-step-by-step.html#starting

Let me know if that doesn't work for you. We can add a comment where it's actually being used.

Reply:
Got it! Thanks for the detailed explanation. Updating now